makefiles/wasmer.mk

```diff
@@ -0,0 +1,70 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS    += wasmer
+WASMER_VERSION := 2.0.0
+DEB_WASMER_V   ?= $(WASMER_VERSION)
+
+wasmer-setup: setup
+	$(call GITHUB_ARCHIVE,wasmerio,wasmer,$(WASMER_VERSION),v$(WASMER_VERSION))
+	$(call EXTRACT_TAR,wasmer-$(WASMER_VERSION).tar.gz,wasmer-$(WASMER_VERSION),wasmer)
+
+ifneq ($(wildcard $(BUILD_WORK)/wasmer/.build_complete),)
+wasmer:
+	@echo "Using previously built wasmer."
+else
+wasmer: wasmer-setup
+	unset CFLAGS && $(DEFAULT_RUST_FLAGS) cargo build \
+		--manifest-path="$(BUILD_WORK)/wasmer/lib/cli/Cargo.toml" \
+		--release \
+		--target=$(RUST_TARGET) \
+		--features="cranelift" \
+		--bin wasmer
+	unset CFLAGS && $(DEFAULT_RUST_FLAGS) cargo build \
+		--manifest-path="$(BUILD_WORK)/wasmer/lib/c-api/Cargo.toml" \
+		--release \
+		--target=$(RUST_TARGET) \
+		--features="cranelift,dylib,staticlib"
+	mkdir -p $(BUILD_STAGE)/libwasmer/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib
```

Comment:
You don't need this for the same reason line 31 works without making a bin folder.

```suggestion

```